src/logstatsexp.jl

```diff
@@ -0,0 +1,38 @@
+"""
+    logmeanexp(A; dims=:)
+
+Computes `log.(mean(exp.(A); dims))`, in a numerically stable way.
+"""
+function logmeanexp(A::AbstractArray; dims=:)
+    R = logsumexp(A; dims)
```

Comment:
Can you try giving everything more descriptive variable names? (Bing AI or GPT-4  are helpful for this IME.)